### ISSUE TITLE
.option('x', { normalize: false }) still normalizes the option 

### ISSUE THREAD

**Issue by brentsnook** [yargs/yargs#947]
Yargs 8.0.2 still normalizes an argument even if a values of `false` is passed to options.

```javascript
#!/usr/bin/env node
"use strict";
const yargs = require('yargs');
yargs.option('database', { normalize: false });
console.log(yargs.argv.database);
```
Notice how the slash in the URL is removed:
```
> ./normalize-test.js --database mongodb://url
mongodb:/url
```
Is this because the code is just checking for the key presence rather than the actual value?
https://github.com/yargs/yargs/blob/a40cbc95277037b614c6a36b817ad31d4327fdae/yargs.js#L594

**Comment by bcoe** [comment]
@brentsnook this sounds like a bug, and I think you've hunted down the root cause. I would happily accept a pull request if you want to see it over the finish line.

**Comment by brentsnook** [comment]
Will try to get to it over the next week or so and send a PR

**Comment by Morishiri** [comment]
As I merged #952 - closing this.
